tutorial/delete-name.md

```diff
@@ -0,0 +1,89 @@
+# Delete Name
+
+## Msg
+
+Now it is time to define the `Msg` for deleting names and add it to the `./x/nameservice/types/msgs.go` file. This code is very similar to `SetName`:
+
+```go
+// MsgDeleteName defines a DeleteName message
+type MsgDeleteName struct {
+	Name  string		 `json:"name"`
```

Comment:
Formatting is off.

Reply:
Didn't quite understand this one. I followed the same format as buy-name.md and set-name.md. Everywhere I have noticed that the `json` names are structured in a column, one after the other. Hence the code looks as follows:
```
type MsgDeleteName struct {
	Name  string         `json:"name"`
	Owner sdk.AccAddress `json:"owner"`
}
```
Since sdk.AccAddress as a term takes more space than string, the term `json:"name"` is pushed to align with `json:"owner"`

Reply:
The formatting of Go is not correct. See: https://github.com/cosmos/sdk-application-tutorial/blob/88709734312254089c3c1e3ff5935b436fce4cba/tutorial/delete-name.md

The field tags need to be aligned. You can just copy-paste this from the actual gocode.

Reply:
Oh I thought you meant the alignment of the field tags in the code is not right. Yeah I couldn't see the preview before pushing the code. I'll copy it from there.

Reply:
On a side note, found the preview option in vscode.